packages/standard-minifiers-css/plugin/minify-css.js

```diff
@@ -14,17 +14,19 @@ CssToolsMinifier.prototype.processFilesForBundle = function (files, options) {
 
   if (! files.length) return;
 
-  var merged = mergeCss(files);
-
+  // Don't merge or minify anything for development
   if (mode === 'development') {
-    files[0].addStylesheet({
-      data: merged.code,
-      sourceMap: merged.sourceMap,
-      path: 'merged-stylesheets.css'
+    files.forEach(function (file) {
+      file.addStylesheet({
+        data: file.getContentsAsBuffer(),
+        sourceMap: file.getSourceMap(),
+        path: file.getPathInBundle()
```

Comment:
You should also move the URL rewriting code ([here](https://github.com/meteor/meteor/blob/b11649f7d469ad14b4b53e6ff1c7de2887ca7580/packages/minifiers-css/minifiers.js#L55)) outside of the merging function so that the served CSS is actually the same in dev and prod.